dataflow/src/main/java/org/checkerframework/dataflow/livevariable/LiveVarNode.java

```diff
@@ -0,0 +1,55 @@
+package org.checkerframework.dataflow.livevariable;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.cfg.node.FieldAccessNode;
+import org.checkerframework.dataflow.cfg.node.LocalVariableNode;
+import org.checkerframework.dataflow.cfg.node.Node;
+
+/**
+ * A LiveVarNode contains a CFG node, which can only be a LocalVariableNode or FieldAccessNode. It
+ * is used to represent the estimate of live variables at certain CFG block during dataflow
+ * analysis. When merging two stores, the `equals` method will perform the value equality check
+ * between two LiveVarNodes.
```

Comment:
The comment about merging stores here seems misplaced - that should be in the LVStore documentation.
Here, the important part is the difference to just using the Node - that equality for the LVNode uses value equality instead of reference equality. Explain why this is done. Then the comment on the equals method can be shorter and just refer to the comment here.

Reply:
Thanks for pointing out! Fixed it.